pkg/docker/index.html

```diff
@@ -152,7 +152,10 @@ <h1 translatable="yes">Can&rsquo;t connect to Docker</h1>
           </tr>
           <tr>
             <td translate>Image:</td>
-            <td colspan="3" id="container-details-image"></td>
+            <td colspan="3">
+            <div id="container-details-image"></div>
+            <div id="container-details-image-id"></div>
```

Comment:
Out of interest (and learning): why is it better to use two different fields here? I would think that we only ever show one of them: if we have a name, we wouldn't show an ID. Wouldn't it be simpler to keep the single field and set `img_name ?: image.Id`?